index.js

```diff
@@ -73,15 +69,14 @@ class LogForwardingPlugin {
 
   /**
    * Makes a Subscription Filter object for given function name
-   * @param  {String} serviceName  name of current service
-   * @param  {String} stage        stage this lambda is being deployed to
    * @param  {String} arn          arn of the lambda to forward to
    * @param  {String} functionName name of function to make SubscriptionFilter for
    * @param  {String} filterPattern filter pattern for the Subscription
    * @return {Object}               SubscriptionFilter
    */
-  static makeSubscriptionFilter(serviceName, stage, arn, functionName, filterPattern) {
-    const logGroupName = `/aws/lambda/${serviceName}-${stage}-${functionName}`;
+  makeSubscriptionFilter(arn, functionName, filterPattern) {
+    const functionObject = this.serverless.service.getFunction(functionName);
+    const logGroupName = this.provider.naming.getLogGroupName(functionObject.name);
     const filter = {};
     filter[`SubscriptionFilter${functionName}`] = {
```

Comment:
@domaslasauskas 

also if function names contain dashes, deploy will fail. 
So instead of directly using `functionName`, there should be `this.provider.naming.getNormalizedFunctionName(functionName)`